### DISCUSSION TITLE
Prettier for the schema

### DISCUSSION THREAD

**Discussion by alan345** [apollographql/apollo-server#7651] [General] [answered]
Our schema is getting pretty big. We have indentation and spacing inconsistency.  
Is there a way to make [prettier](https://prettier.io/) (or any other formatting library) works with the keyword  `#graphql?

Full code here: https://www.apollographql.com/docs/apollo-server/getting-started/#step-3-define-your-graphql-schema
```
import { ApolloServer } from '@apollo/server';
import { startStandaloneServer } from '@apollo/server/standalone';

// A schema is a collection of type definitions (hence "typeDefs")
// that together define the "shape" of queries that are executed against
// your data.
const typeDefs = `#graphql
  # Comments in GraphQL strings (such as this one) start with the hash (#) symbol.

  # This "Book" type defines the queryable fields for every book in our data source.
  type Book {
    title: String
    author: String
  }

  # The "Query" type is special: it lists all of the available queries that
  # clients can execute, along with the return type for each. In this
  # case, the "books" query returns an array of zero or more Books (defined above).
  type Query {
    books: [Book]
  }
`;
```

**Comment by trevor-scheer** [comment, marked as answer]
Interesting predicament, hadn't considered it when we started pushing this recommendation. Prettier recognizes the `gql` tag though, so you can either use the real thing (and pass a `DocumentNode` to Apollo Server instead of a `string`) or fake it like so:

```ts
const gql = String.raw;

const typeDefs = gql`
  ...
`;

**Comment by glasser** [comment]
Alternatively (perhaps preferably), keep it in its own .graphql file (and load it with fs.readFileSync or something).
